src/common/chains.ts

```diff
@@ -3,9 +3,3 @@ export enum SupportedChainId {
   GOERLI = 5,
   GNOSIS_CHAIN = 100,
 }
-
-export const ALL_SUPPORTED_CHAIN_IDS: SupportedChainId[] = [
-  SupportedChainId.MAINNET,
-  SupportedChainId.GOERLI,
-  SupportedChainId.GNOSIS_CHAIN,
-]
```

Comment:
This was a public export.
Won't it break anything?
Might add a deprecation warning and remove on next version?

Reply:
It's still here, exported from `src/index.ts`